src/components/CustomDropdown/DropdownMenuItems.tsx

```diff
@@ -0,0 +1,144 @@
+import React from "react";
+import { Link, LinkProps } from "react-router-dom";
+import styled, { css } from "styled-components";
+import { ArrowRight } from "../Icons";
+
+// Dropdown items can be of a few component
+// varieties, including buttons, router links, and
+// anchor tags. This file unifies their styling,
+// and then divides them into three component exports
+// for semantically explicit usage in dropdowns.
+
+// Common styles
+const baseStyles = css`
+    font-family: ${(props) => props.theme.typography};
```

Comment:
this is our only styled component for now, but we will probably want to add a `GlobalStyles` provider to get things like basic fonts into all styled components, and it could potentially replace some or all of the global style rules in style.css if we do that.